src/BenchmarksDriver/Program.cs

```diff
@@ -107,6 +107,9 @@ public static int Main(string[] args)
             var kestrelThreadCountOption = app.Option("--kestrelThreadCount",
                 "Maps to KestrelServerOptions.ThreadCount.",
                 CommandOptionType.SingleValue);
+            var kestrelThreadPoolDispatchingOption = app.Option("--kestrelThreadPoolDispatching",
+                "Maps to InternalKestrelServerOptions.ThreadPoolDispatching.",
```

Comment:
This should mention `InternalKestrelServerOptions.ThreadPoolDispatching` is a `bool` that defaults to `true`.